src/App.tsx

```diff
@@ -1,24 +1,246 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import {
+  getTodos,
+  addTodo,
+  deleteTodo,
+  completeTodo,
+  editTodo,
+} from './api/todos';
+import { Filter } from './types/Filter';
+import { ErrorTypes } from './types/ErrorTypes';
+import {
+  Header, TodosList, Footer, ErrorNotifications,
+} from './components';
 
-const USER_ID = 0;
+const USER_ID = 6335;
 
 export const App: React.FC = () => {
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<ErrorTypes | null>(null);
+  const [filter, setFilter] = useState<Filter>(Filter.All);
```

Comment:
redundant type. it will be Filter by default because of default state
```suggestion
  const [filter, setFilter] = useState(Filter.All);
```

Reply:
check bellow states as well